Gemfile

```diff
@@ -28,22 +28,14 @@ group :system_tests do
   gem 'beaker-puppet_install_helper', :require => false
 end
 
-is_x64 = Gem::Platform.local.cpu == 'x64'
-if is_x64
-  platform(:x64_mingw) do
-    gem "win32-dir", "~> 0.4.9", :require => false
-    gem "win32-process", "~> 0.7.4", :require => false
-    gem "win32-service", "~> 0.8.4", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-  end
-else
-  platform(:mingw) do
-    gem "win32-process", "~> 0.6.5", :require => false
-    gem "win32-service", "~> 0.7.2", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-    if RUBY_VERSION =~ /^1\./ then
-      gem "win32console", :require => false
-    end
+platforms :mswin, :mingw do
```

Comment:
@joshcooper 

Puppet controls the actual dependencies once you are above 3.5.x, not this gemfile.
```
    puppet (4.2.0-x64-mingw32)
      facter (> 2.0, < 4)
      ffi (~> 1.9.6)
      hiera (>= 2.0, < 3)
      json_pure
      minitar (~> 0.5.4)
      win32-dir (~> 0.4.9)
      win32-eventlog (~> 0.6.2)
      win32-process (~> 0.7.4)
      win32-security (~> 0.2.5)
      win32-service (~> 0.8.6)
```